mungegithub/github/github.go

```diff
@@ -72,6 +72,14 @@ var (
 	combinedStatusLifetime = 5 * time.Second
 )
 
+func suggestOauthScopes(resp *github.Response, err error) error {
+	if resp != nil && resp.StatusCode == http.StatusForbidden {
+		oauthScopes := resp.Header.Get("X-Accepted-OAuth-Scopes")
+		err = fmt.Errorf("%v - you need to use the following oauth scopes: %s", err, oauthScopes)
```

Comment:
Also technically it lists all scopes that would work, so the language here should be `"use one of the following..."`